CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+add rpm stuff (spec file)
```

Comment:
Issue #6 should be mentionced. "add" -> "Add"

Reply:
CNR Updated in https://github.com/telefonicaid/sigfox-iotagent/pull/32/commits/e4426f8430cb50253af25f405db03638e5c7a27b